### ISSUE TITLE
'NoneType' object has no attribute 'best_model'

### ISSUE THREAD

**Issue by joaoabr** [pharmpy/pharmpy#1063]
run_amd leads to the error **AttributeError: 'NoneType' object has no attribute 'best_model'**. The start model, structural and iivsearch seem to run fine. The tool seems to crash with resmod - see below. Let me know if you want the files and folders. 

```
order <- c('structural', 'iivsearch', 'residual', 'iovsearch', 'allometry', 'covariates')
res <- run_amd(input=dataset_path,
               modeltype='pk_iv', vc_init = V, cl_init = CL,
               order=order,
               categorical=c('APGR'),
               continuous=c('WGT'),
               allometric_variable=c("WGT"))

```
Leads to the following messages:

```
distributed.utils_perf - WARNING - full garbage collections took 11% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 11% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 11% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 11% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 12% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 12% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 12% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 13% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 13% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 13% CPU time recently (threshold: 10%)
distributed.utils_perf - WARNING - full garbage collections took 13% CPU time recently (threshold: 10%)
Error in py_call_impl(callable, dots$args, dots$keywords) : 
  AttributeError: 'NoneType' object has no attribute 'best_model'

Detailed traceback:
  File "C:\PROGRA~1\Python39\lib\site-packages\pharmpy\tools\run_amd.py", line 161, in run_amd
    next_model = subresults.best_model
```

For resmod:

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/abrantj1/AppData/Local/Temp/12/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/abrantj1/AppData/Local/Temp/12/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

summary_tool |   |   |   |  
-- | -- | -- | -- | --
model | description | dofv | ofv | rank | parent_model
base_model | [CL,VC] | 0 | 689.8828 |   | base_model

</body>

</html>

However:

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/abrantj1/AppData/Local/Temp/12/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/abrantj1/AppData/Local/Temp/12/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

models |   |   |   |  
-- | -- | -- | -- | --
model | dvid | iteration | dofv | parameters
IIV_on_RUV | 1 | 1 | 5.913563 | {'omega': 0.096932}
combined | 1 | 1 | 3.229677 | {'sigma_add': 155.616, 'sigma_prop': 0.673874}
power | 1 | 1 | 2.68797 | {'theta': -0.27596}
time_varying1 | 1 | 1 | 0.809679 | {'theta': 0.862638}
time_varying2 | 1 | 1 | 1.712172 | {'theta': 1.16238}
time_varying3 | 1 | 1 | 1.176642 | {'theta': 1.1417}

</body>

</html>

**Comment by stellabelin** [comment]
Hi! It doesn't work for me to read in and not sure what the issue is, would it be possible to email me the file instead?

**Comment by joaoabr** [comment]
done

**Comment by pharmpy-dev-123** [comment]
Might be fixed by #1065.

**Comment by pharmpy-dev-123** [comment]
#1065 was merged into `main`. Should be included in release `0.76.1` or `0.77.0`.

**Comment by ZheHuang96** [comment]
Should I close this issue or wait until we have new release? @pharmpy-dev-123

**Comment by pharmpy-dev-123** [comment]
Let's wait until we have a release.

**Comment by pharmpy-dev-123** [comment]
Included in release `0.76.5`. Closing.
